Homework-1/index.js

```diff
@@ -0,0 +1,84 @@
+function loadReviews() {
+  if (!localStorage.getItem("reviews"))
+    localStorage.setItem("reviews", JSON.stringify([]));
+  const reviews = JSON.parse(localStorage.getItem("reviews"));
+  calculateAverage(reviews ? reviews : []);
+
+  document.getElementsByClassName("reviews")[0].innerHTML = "";
```

Comment:
You are using document.getElementsByClassName("reviews")[0] two times in this function. It would be useful to assign it to a variable and use it that way. This way if for some reason you decide to change the element id from 'reviews' to 'reviews-container' or something similar, you would have to update your code in only one place. This will help you minimise bugs.